be/src/olap/rowset/beta_rowset_writer.cpp

```diff
@@ -131,7 +131,8 @@ Status BetaRowsetWriter::add_block(const vectorized::Block* block) {
     return _segment_creator.add_block(block);
 }
 
-Status BetaRowsetWriter::_generate_delete_bitmap(int32_t segment_id) {
+Status BetaRowsetWriter::_generate_delete_bitmap(int32_t segment_id,
+                                                 bool is_unique_key_ignore_mode) {
```

Comment:
it's not necessary to add these parameters, you can get the tablet schema inside `calc_segment_delete_bitmap`.